tests/ContainerTest.php

```diff
@@ -23,8 +23,10 @@ public function testSetsAndGetsSimplePrototypedClosure()
 
         $this->assertEquals($container->get('test', ['hello']), 'hello');
         $this->assertEquals($container->get('test', ['world']), 'world');
+        $this->assertEquals($container->get('test', [true]), true);
+        $this->assertEquals($container->get('test', [false]), false);
     }
-
+    
```

Comment:
this isn't under CS test, but seems a leftover from some IDE adding useless spaces :-)